bin/overview-modules/0_overview-heat-flow-module.ipynb

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
<blockquote>Upon completion of this module you would know how to use python to:</blockquote><p>Replace "would" with "will"?</p><p> </p><blockquote>An introduction to the conductive heat transfer mechanism within the Earth.</blockquote><p>Suggestion: "An introduction to conductive heat transfer within the Earth."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>